app/konnect/vitals/overview-dashboard.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Overview Dashboard
+no_version: true
+content_type: reference
+---
+
+The Overview Dashboard provides metrics for Services catalogued by Service Hub within a selected time interval for the following two categories: 
+
+* **Traffic** - total number of HTTP requests.
+* **Errors** - percentage of failed HTTP requests. 
```

Comment:
```suggestion
* **Traffic**: Total number of HTTP requests.
* **Errors**: Percentage of failed HTTP requests. 
```
I'd go with colons for consistency's sake. That's the style we use most frequently in the kong docs.